install/.colcon_install_layout

```diff
@@ -0,0 +1 @@
+isolated
```

Comment:
@Abishalini Please remove this directory

Reply:
I have removed the log directories and updated ./colcon_install_layout